build/configs/qemu/README.md

```diff
@@ -4,7 +4,9 @@ QEMU is a generic and open source machine emulator and virtualizer.
 
 ## Information
 
-QEMU uses lm3s6963-ek machine.
+QEMU and TizenRT support the lm3s6965-ek board.  
```

Comment:
Tizen RT can be configured and built for lm3s6965-ek board which is guranteed to run ONLY on QEMU emulator.